Scripts/QRadarGetCorrelationLogs/QRadarGetCorrelationLogs.py

```diff
@@ -1,86 +1,106 @@
 import demistomock as demisto
 from CommonServerPython import *
-QUERY = "select RULENAME({0}), sourceip, destinationip, eventcount, sourceport, username, " \
-        "starttime, destinationport, magnitude, identityip, CATEGORYNAME(category), " \
-        "PROTOCOLNAME(protocolid), LOGSOURCENAME(logsourceid){1} from " \
-        "events where \"CRE Name\" IS NULL AND INOFFENSE({2}) START '{3}'"
-
-d_args = demisto.args()
-
-offense_id = demisto.get(d_args, 'offenseID')
-start_time = demisto.get(d_args, 'startTime')
-correlation_id = demisto.get(d_args, 'qid')
-additional_query_fields = demisto.get(d_args, 'additionalQueryFields')
-# Try converting from date string to timestamp
-try:
-    start_time = date_to_timestamp(str(start_time), '%Y-%m-%dT%H:%M:%S.%f000Z')
-except Exception:
-    pass
-
-d_args["query_expression"] = QUERY.format(correlation_id, additional_query_fields, offense_id, start_time)
-
-resp = demisto.executeCommand('QRadarFullSearch', d_args)
-if isError(resp[0]):
-    demisto.results(resp)
-else:
-    data = demisto.get(resp[0], 'Contents.events')
 
-    if not data:
-        resp[0]['HumanReadable'] = "No logs were found for correlation id {0}".format(correlation_id)
-    else:
-        data = data if isinstance(data, list) else [data]
 
-        QRadar = {
-            'Log': []
-        }  # type: Dict
+def get_query(cre_name_null):
+    if cre_name_null == "True":
+        query = "select RULENAME({0}), sourceip, destinationip, eventcount, sourceport, " \
+                "username, starttime, destinationport, magnitude, identityip, CATEGORYNAME(category), " \
+                "PROTOCOLNAME(protocolid), LOGSOURCENAME(logsourceid){1} from " \
+                "events where \"CRE Name\" IS NULL AND INOFFENSE({2}) START '{3}'"
+    else:
+        query = "select RULENAME({0}), sourceip, destinationip, eventcount, " \
+                "sourceport, username, starttime, destinationport, magnitude, " \
+                "identityip, CATEGORYNAME(category), PROTOCOLNAME(protocolid), " \
+                "LOGSOURCENAME(logsourceid){1} from events " \
+                "where \"CRE Name\" <> NULL AND INOFFENSE({2}) START '{3}'"
+    return query
+
+
+def main():
+    d_args = demisto.args()
+    is_cre_name_null = demisto.args().get("is_cre_name_null", "True")
+    QUERY = get_query(is_cre_name_null)
```

Comment:
there is no need to QUERY be with upper case

Reply:
It was like that before and I'm not refactoring the script, just re wrote it to be more easily testable